tools/SetupFlow/DevHome.SetupFlow.UnitTest/LoadingScreenTests.cs

```diff
@@ -11,7 +13,7 @@ public class LoadingScreenTests : BaseSetupFlowTest
     [TestMethod]
     public void HideRetryBannerTest()
     {
-        var loadingViewModel = new LoadingViewModel(StringResource.Object, null, TestHost);
+        var loadingViewModel = new LoadingViewModel(StringResource.Object, TestHost!.GetService<SetupFlowOrchestrator>(), TestHost);

```

Comment:
```suggestion
        var loadingViewModel = TestHost!.GetService<LoadingViewModel>();
```

All services will be auto injected 😉 `LoadingViewModel` should also be registered in the test host container.